packages/webapp/pages/posts/[id].tsx

```diff
@@ -94,26 +86,32 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
   });
 
   useEffect(() => {
-    window.addEventListener('popstate', () => {
+    globalThis.window?.addEventListener('popstate', () => {
       router.reload();
     });
   }, []);
```

Comment:
We actually need a cleanup here.
Did we not introduce a `useWindow` kind of hook?

Reply:
Or useEventListener, not sure what we called it.